packages/marketplace-ui/src/components/admin/LaunchPreview.tsx

```diff
@@ -0,0 +1,72 @@
+import {
+  Image,
+  Stack,
+  Text,
+  Flex,
+  Icon,
+  useDisclosure,
+} from "@chakra-ui/react";
+import React from 'react';
+import { PublicKey } from '@solana/web3.js';
+import { useMetaplexTokenMetadata, useReserveAmount, useTokenSwapFromId } from "@strata-foundation/react";
+import { MdSettings } from "react-icons/md";
+import { LaunchSettingsModal } from "./LaunchSettingsModal";
+import { useRouter } from "next/router";
+import { routes, route } from "../../utils/routes";
+
+
+interface LaunchPreviewProps {
+  id: PublicKey | undefined;
+  name: string | undefined;
+  image: string | undefined;
+}
+
+export const LaunchPreview = ({ id, name, image }: LaunchPreviewProps) => {
+  const { tokenBonding } = useTokenSwapFromId(id);
+  const { isOpen, onToggle, onClose } = useDisclosure();
+  const reserveAmount = useReserveAmount(tokenBonding?.publicKey);
+  const { metadata } = useMetaplexTokenMetadata(tokenBonding?.baseMint);
+  const router = useRouter();
+  return (
+    <Flex alignItems="center" w="full">
+      <Image
+        alt="Token logo"
+        marginLeft="2em"
+        w="50px"
+        h="50px"
+        borderRadius="50%"
+        src={image}
+      />
+      <Stack 
+        paddingLeft="10px" 
+        onClick={() => {
+          router.push(
+            route(routes.swap, {
```

Comment:
This should go to the admin settings page in docs. You should add one for fully managed tokens, I don't think we have that yet.